_episodes/02-markdown.md

```diff
@@ -24,9 +24,220 @@ keypoints:
+> 
+> 23. enumerations
+> 23. starting at any number
+> 23. continue to increase
+> 26. in steps of one.
+>     1. Sub enumerations (in some editors have to start at **one**)
+>     2. and can include
+>        - sub lists
+>     4. keeping the enumeration count.
+>
+> > ## Solution
+> > ~~~
+> > - A list of items
+> > - can be
+> >   - deeply
+> >     - nested
+> > - returning to the top-level like this.
+> > 
+> > New text blocks require **blank lines** and 
+> > 
+> > 23. enumerations
+> > 23. starting at any number
+> > 23. continue to increase
+> > 26. in steps of one.
+> >     1. Sub enumerations (in some editors have to start at **one**)
+> >     2. and can include
+> >        - sub lists
+> >     4. **without** disturbing the enumeration count.
+> > ~~~
+> > {: .markdown }
+> {: .solution }
+{: .challenge }
+
+
+
+
+> ## Add Code
+> Fill in the blanks below to create this rendered view
+> 
+> Discussing code details can be very helpful.  
+> Therefore, referring to variables, e.g. `path` in a formatted way is great.
+>
+> Larger code snippets look like this:
+> ```
+> print('Hello World')
+> ```
+>
+> ~~~
+> Discussing code details can be very helpful.  
+> Therefore, referring to variables, e.g. _path_ in a formatted way is great.
+>
+> Larger code snippets look like this:
+>
+> ___
+> msg = 'Hello World'
+> print(msg)
+> ___
+> ~~~
+> {: .source }
+> 
+> > ## Solution
+> > ~~~
+> > Discussing code details can be very helpful.  
+> > Therefore, referring to variables, e.g. `path` in a formatted way is great.
+> >
+> > Larger code snippets look like this:
+> > ```
+> > print('Hello World')
+> > ```
+> > ~~~
+> > {: .markdown }
+> {: .solution }
+{: .challenge }
+
+
+> ## Syntax Highlighting
+> Syntax highlighting increases readability a lot.  
+> Guess in which programming language the following code snippets are written
+> and fill the gaps to activate syntax highlighting creating this rendered view
+> 
+> -------------
+> ```R
+> print(paste("How","are","you?"), quote = FALSE)
+> ```
+> -------------
+> 
+> ```python
+> s = "How are you?"
+> print s
+> ```
+> -------------
+> 
+> ```html
+> <!DOCTYPE html>
+> <html>
+> <body>
+> > <a href=https://carpentries.org/>This is a link</a>
+> > </body>
+> </html>
+> ```
+> -------------
+>
+> ~~~
+> ```_
+> print(paste("How","are","you?"), quote = FALSE)
+> ```
+> 
+> ```______
+> s = "How are you?"
+> print s
+> ```
+> 
+> ```____
+> <!DOCTYPE html>
+> <html>
+> <body>
+> > <a href=https://carpentries.org/>This is a link</a>
+> > </body>
+> </html>
+> ```
+> ~~~
+> {: .source }
+>
+> > ## Solution
+> > ~~~
+> > ```R
+> > print(paste("How","are","you?"), quote = FALSE)
+> > ```
+> > 
+> > ```python
+> > s = "How are you?"
+> > print s
+> > ```
+> > 
+> > ```html
+> > <!DOCTYPE html>
+> > <html>
+> > <body>
+> > > <a href=https://carpentries.org/>This is a link</a>
+> > > </body>
+> > </html>
+> ```
+> > ~~~
+> > {: .markdown }
+> {: .solution }
+{: .challenge }
+
+
+
+
```

Comment:
It would be great to add one more exercise about links here, perhaps a fill-in-the-blanks? No need to do that now, though, we can open an Issue/include a TODO to remind us later.

Reply:
Opened an [issue](https://github.com/tobyhodges/building-websites-with-jekyll-and-github-or-gitlab/issues/4#issue-668666187) and will resolve it.